charts/postgres-db-sync/templates/sync-job.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "pgdbsync.fullname" . }}
+spec:
+  schedule: "{{ $.Values.cronjob.schedule }}"
+  concurrencyPolicy: "Forbid"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: postgres-to-postgres-sync-cronjob
+            image: {{ printf "%s:%s" $.Values.image.repository $.Values.image.tag }}
+            imagePullPolicy: {{ $.Values.image.pullPolicy }}
+            command: 
+              - "/bin/bash"
+              - "-c"
+              - |
+                /bin/bash <<'EOF'
+
+                {{- /* Iterate through each table mapping (Input table -> Output table) */}}
+                {{- range $mapping := $.Values.tableMappings }}
```

Comment:
Way better ! I still don't like having two languages together but that's fine for now.